_pages/locations/chicago.md

```diff
@@ -172,4 +172,8 @@ Recycling, including battery recycling, is available on the 2nd floor of our bui
 
 Dollop and Intelligentsia are the current fan favorites for coffee. If you like darker roasts, Lavazza may be up your alley.
 
+## How do people dress?
 
+Feel free to dress how you like! If you wear a T-shirt and jeans, that's fine. Sometimes folks wear hoodies. If someone has a meeting with agency folks, they may dress up more. We care more that you are comfortable and confident as yourself rather than following someone’s particular rules of formality.
```

Comment:
Maybe mention, even on your first day, no one's going to look at you funny if you show up in a hoodie? Maybe?

FWIW, I wore a sport coat and felt slightly overdressed, but not uncomfortably so.

Reply:
@tbaxter-18f Hm, I think the current text mentions a couple of times that it's OK to dress the way you like. I don't necessarily think we need to add additional copy about that. However, if you open a PR into this PR with suggested text changes, I will absolutely merge those in! Or honestly feel free to commit directly on this branch if you feel so moved.

Reply:
Nah. Just a suggestion. If you think we're covered, 👍